app/blog/2017-08-03-speaker-spotlight-chad.md

```diff
@@ -0,0 +1,86 @@
+
+**How would you make the Web better?**
+
+That's easy: JavaScript modules. We're now 2 years past the formalization of the ES2015 spec and browsers have just barely begun to support them. The pace towards support has moved a glacial speed which has been frustrating. In addition, developers have openly embraced the ability to resolve modules from the Node ecosystem, but browsers use a different algorithm. This struggle between the algorithm that build tools use and browsers use hasn't been fun for developers who feel very much caught in the middle.
+
+**What’s an example of working in the web dev community that delights you?**
+
+One of the challenges is the wide variety of developer skillsets and abilities. You can have a designer who just starts tinkering with HTML and CSS in the same group as a true JavaScript engineer who is architecting large, scalable applications that have to perform well in a large variety of conditions.
+
+As a Engineering manager, that makes hiring very challenging!
+
+But I love the speed at which a new participant into that community can actively contribute and build something truly usable. However at the same time I feel like the knowledge required to be at that top tier of developer is increasing so fast that we are leaving a large set of developers behind.
+
+Being able to make visual changes to the app without having to be deeply involved in the JavaScript is one of the greatest benefits of Polymer. Because you are working with HTML and CSS directly and changes are isolated to that component alone, it's easy for someone who specializes in HTML/CSS to make visual corrections or changes without having to really know the framework internals or interfere with the business logic.
+
+**Just two more questions for you - 1. What is your favourite food and 2. What are you looking forward to in your visit to Copenhagen?**
+
+1. I'm not much of a favorites type of person - I get bored with the same thing over and over. Anything quality I enjoy.
+
+2. I love traveling, so I'm always looking forward to seeing a new place. But I'm also very excited to spend in-person time with the Google developers I collaborate with so frequently.
+
+
+-----
+
+This year's summit takes place August 22–23 at [Lokomotivværkstedet](http://www.lvcph.dk/index-eng.html) in beautiful downtown [Copenhagen, Denmark](https://goo.gl/maps/pgFPsEkRRcS2).
+
+Like previous years, we'll have talks and codelabs from the Polymer team, plus food, fun, and plenty of space for informal conversations.
+
+For the first time this year, we'll be asking for talks from community members, so we can hear from you, too.
+
+To be notified when registration opens, sign up with your email address at [g.co/polymersummit2017](http://g.co/polymersummit2017). As always, tickets are  free.
```

Comment:
Since registration is open, we should probably say something different here. We should probably check with @wmginsberg or @tjsavage on the appropriate messaging here...